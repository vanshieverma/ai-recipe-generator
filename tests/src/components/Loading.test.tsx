import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import Loading from "../../../src/components/Loading";
 
describe('The Loading spinner', () => {
  it('renders', () => {
    const { container } = render(<Loading />)
    expect(container).toMatchSnapshot()
  })
})