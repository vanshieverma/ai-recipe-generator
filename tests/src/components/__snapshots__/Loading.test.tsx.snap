// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Loading spinner renders 1`] = `
<div>
  <div
    class="flex items-center justify-center mt-5"
  >
    <div
      class="relative"
    >
      <div
        class="h-24 w-24 rounded-full border-t-8 border-b-8 border-gray-200"
      />
      <div
        class="absolute top-0 left-0 h-24 w-24 rounded-full border-t-8 border-b-8 border-blue-500 animate-spin"
      />
    </div>
  </div>
</div>
`;
